Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -115,6 +115,7 @@
 
 #if defined(HAVE_XRANDR) && HAVE_XRANDR
 #include "UICommon/X11Utils.h"
+#undef None
```

Comment:
I wonder if this should be commented; so we know where it comes from (as a reminder that we might be able to remove it later at some point). Thoughts?

Reply:
I added a comment.